addons/core/tests/integration/components/filter-tags/index-test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { setupIntl } from 'ember-intl/test-support';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import sinon from 'sinon';
+
+module('Integration | Component | filter-tags/index', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  test('it renders', async function (assert) {
+    const router = this.owner.lookup('service:router');
+    sinon.stub(router, 'currentRoute').value({
```

Comment:
You still need an afterEach hook to call `sinon.restore()`

Reply:
Right, thanks for the reminder.